src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.util.Limelight;
+import edu.wpi.first.math.geometry.Pose3d;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+public class Vision extends DashboardedSubsystem {
```

Comment:
Once the camera server works it should be in here.
maybe try this code (taken from rapid react 2022, idk if it actually does something):
`CameraServer.startAutomaticCapture();`
`CvSink cvSink = CameraServer.getVideo();`
`CvSource outputStream = CameraServer.putVideo("back camera", 720, 1280);`